### PR TITLE
Architecture and feature docs3

### PR DESCRIPTION
**Pull request by indie**
Final updates for architecture and feature docs including different wire diagrams requested by Harry, cleaner tables on Quick Start and replace dummy table data with ref to nbench for performance metrics

### CODE REVIEW

File: ecosystem-overview.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+## How do I connect a framework?
+
+The nGraph Library manages framework bridges for some of the more widely-known 
+frameworks. A bridge acts as an intermediary between the nGraph core and the 
+framework, and the result is a function that can be compiled from a framework. 
+A fully-compiled function that makes use of bridge code thus becomes a "function 
+graph", or what we sometimes call an nGraph graph.
+
+**Note** Low-level nGraph APIs are not accessible dynamically via bridge code;
```
Comment (diyessi) [inline ecosystem-overview.md]: I don't know what this is trying to say, but it's currently wrong.

File: .gitignore [hunk 2]
```diff
@@ -93,6 +93,10 @@ doc/doxygen/latex/*
 doc/doxygen/xml/*
 doc/doxygen/html/*
 
+# Cmake artifacts
+
+/CMakeFiles/
```
Comment (rkimballn1) [inline .gitignore]: This is not necessary. There is a bug that will be addressed by a separate PR and if you try to do an in-tree build you will have more than just these files polluting the source tree.

File: ecosystem-overview.md (hunk 1 above)
Comment (indie) [reply ecosystem-overview.md]: okay, this whole paragraph is getting deleted.

File: .gitignore (hunk 2 above)
Comment (indie) [reply .gitignore]: Gotcha, reverted.

File: doc/sphinx/source/frameworks/validation-testing.rst [hunk 3]
```diff
@@ -4,97 +4,86 @@
 Validation and testing
 ######################
 
-* **Validating** -- To provide optimizations with nGraph, we first 
-  confirm that a given workload is :term:`validated` as being functional; 
-  that is, we can successfully load its serialized graph as an nGraph 
-  :term:`function graph`. Following here is a list of 14 workloads 
-  we've tested with success.
+We validated perfomance for the following TensorFlow and MXNet workloads:
```
Comment (indie) [inline doc/sphinx/source/frameworks/validation-testing.rst]: Performance is typo.  Also, since this is the first instance of TF and MXNet on this page, probably need the asterisk
